docs/conf.py.in

```diff
@@ -74,3 +74,73 @@ cpp_index_common_prefix = ["cvc5::api::"]
 
 # where to look for include-build-file
 ibf_folders = ['${CMAKE_CURRENT_BINARY_DIR}']
+
+# -- SMT-LIB syntax highlighting ---------------------------------------------
+from pygments.lexer import RegexLexer
+from pygments import token
```

Comment:
I'm glad you added this lexer! However, given that the list of tokens will probably grow quite a bit, can we move it to a separate file?

Reply:
This is not worth the headache in my opinion. Requires copying the file into the build directory and so on.

Reply:
We could put it into the `ext` directory, which is put into the path anyway.

Reply:
Oh nice, that works! Updated.